pytorch_blade/src/torch-mlir/lib/Dialect/TorchConversion/Transforms/DiscDecomposeComplexOps.cpp

```diff
@@ -214,6 +214,8 @@ class DiscDecomposeComplexOpsPass
     patterns.add<ConvertAtenOp<AtenNllLossForwardOp>>(context);
     target.addIllegalOp<AtenNllLossForwardOp>();
 
+    target.addLegalOp(OperationName("torch.operator", context));
```

Comment:
Nit: remove Line 206 has the same effect.

Reply:
done

Reply:
Nit: `target.addLegalOp<TorchOperator>();`

Reply:
done